lahja/asyncio/endpoint.py

```diff
@@ -141,10 +187,16 @@ def __init__(
                 return
 
             if isinstance(message, Broadcast):
-                self.new_msg_func(message)
+                await self.new_msg_func(message)
```

Comment:
This change is effctively changing us from using `Queue.put_nowait` to `Queue.put` which seems more appropriate (but I don't really know why).  Thoughts?